partner_stage/models/res_partner.py

```diff
@@ -14,8 +14,9 @@ def _get_default_stage_id(self):
         )
 
     @api.model
-    def _read_group_stage_id(self, states, domain, order):
-        return states.search([])
+    def _read_group_stage_id(self, stages, domain, order=None):
+        order = order or "sequence, id"
+        return stages.search([], order=order)
```

Comment:
Even simpler, sparing 1 LOC?
```suggestion
        return stages.search([], order=order or "sequence, id")
```